game/templates/game/board.html

```
@@ -537,40 +411,84 @@
+                    </div>
+
+                    <!-- Quick Move Input (SAN) -->
+                    <div class="san-input-container">
+                        <input type="text" id="sanMoveInput" class="san-move-input" placeholder="Move (e.g. e4, Nf3, O-O)" aria-label="Enter move notation" autocomplete="off">
+                        <button type="button" id="sanMoveBtn" class="btn btn-gold san-move-btn">Move</button>
+                    </div>
+                    <div id="sanMoveError" class="san-move-error" role="alert">Invalid notation</div>
+
+                    <div style="margin-top: 10px;">
+                        <div style="display: flex; flex-direction: column; gap: 4px;">
+                            <input type="text" id="manualMoveInput" placeholder="Move (e.g. e2e4)" style="padding: 10px; border-radius: 5px; border: 1px solid #444; background: #1a1a2e; color: #fff; font-size: 0.9rem;" maxlength="5">
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)